src/components/Project/ContentTab/Views/DataView.vue

```diff
@@ -1,55 +1,89 @@
 <template>
-  <div class="data-view-wrapper">
+  <div ref="datawrapper" class="data-view-wrapper h-full">
     <div class="refresh-data">
-      <!-- <Button icon="pi pi-refresh" /> -->
       <Button label="Refresh" icon="pi pi-refresh" />
     </div>
-    <DataTable
-      :value="data"
-      striped-rows
-      responsive-layout="scroll"
-      :scrollable="true"
-      scroll-height="415px"
-    >
-      <Column field="year" header="Year"></Column>
-      <Column field="brand" header="Brand"></Column>
-      <Column field="color" header="Color"></Column>
-    </DataTable>
+    <div id="data_view_cm" class="shadow-3">
+      <codemirror
+        :model-value="data"
+        placeholder="Code goes here..."
+        :style="{ height: '60vh' }"
+        :autofocus="true"
+        :indent-with-tab="true"
+        :tab-zize="2"
+        :extensions="extensions"
+        @change="handleChange"
+      />
+    </div>
   </div>
 </template>
 
 <script>
-  import DataTable from "primevue/datatable"
-  import Column from "primevue/column"
+  import { Codemirror } from "vue-codemirror"
+  import GRAPH_DATA from "@/components/Transformer/D3/miserables.json"
```

Comment:
Can we get data from the API and remove this file all together, this can be saved into a query somewhere, but no need for the file or it’s reading from project folder.